Runtime error on add to project page if user is part of openshift namespace

This happens when we build the namespace menu on the "Deploy Image" tab since we insert "openshift" always. The same item appears twice.

```
angular.js:11706 Error: [ngRepeat:dupes] Duplicates in a repeater are not allowed. Use 'track by' expression to specify unique keys. Repeater: namespace in $select.items, Duplicate key: string:openshift, Duplicate value: openshift
http://errors.angularjs.org/1.3.20/ngRepeat/dupes?p0=namespace%20in%20%24select.items&p1=string%3Aopenshift&p2=openshift
    at angular.js:63
    at ngRepeatAction (angular.js:24772)
    at Object.$watchCollectionAction [as fn] (angular.js:14279)
    at Scope.$digest (angular.js:14412)
    at Scope.$apply (angular.js:14675)
    at done (angular.js:9725)
    at completeRequest (angular.js:9915)
    at XMLHttpRequest.requestLoaded (angular.js:9856)
```

--- a/app/scripts/directives/istagSelect.js
+++ b/app/scripts/directives/istagSelect.js
@@ -29,7 +29,11 @@ angular.module("openshiftConsole")
         $scope.isNamesByNamespace = {};
 
         DataService.list("projects", {}, function(projectData) {
-          $scope.namespaces = ['openshift'].concat(_.keys(projectData.by('metadata.name')).sort());
+          var sortedNamespaces = _.keys(projectData.by('metadata.name')).sort();
+
+          // Use _.uniq to avoid adding "openshift" twice if the user is a
+          // member of the openshift namespace.
+          $scope.namespaces = _.uniq(['openshift'].concat(sortedNamespaces));
 
           // Fetch image streams when a new namespace is picked.
           $scope.$watch('istag.namespace', function(namespace) {

